src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/Offender.kt

```diff
@@ -71,6 +83,22 @@ class OffenderBuilder(
     .also { it.rootOffenderId = it.id }
     .also { offender = it }
 
+  private fun buildAlias(
```

Comment:
I was expecting this to be in a different file for the AliasDsl... is that because Alias is the same as Offender?

Reply:
Yes - a sort of special case. And Offender has an alias which is really just another Offender - so really you are creating an offender from within an offender.

But that is confusing so called it "alias" which is closer to what we know it as